gulp/scripts.js

```diff
@@ -29,14 +31,23 @@ module.exports = {
         sourcemap: isDev ? true : false,
       });
 
-      const donateBundle = await rollup.rollup(rollupConfig({ isDonate: true }));
+      const donateBundle = await rollup.rollup(rollupConfig({ entry: projectRoot('src/scripts/donate.js') }));
 
       await donateBundle.write({
         file: donateOutput,
         format: 'iife',
         name: 'donate_scripts',
         sourcemap: isDev ? true : false,
       });
+      
+      const actionkitBundle = await rollup.rollup(rollupConfig({ entry: projectRoot('src/scripts/giraffe-actionkit.js') }));
```

Comment:
let's just make this `actionkit.js` -- we're in the giraffe repository.  as for the *output* file, I'm more comfortable with that being `giraffe-actionkit.js` to make it clear that it's not one of the many different actionkit.js files floating around.

Reply:
I've renamed the entry file :)